README.md

```diff
@@ -120,6 +120,69 @@ import utc from '@momentjs/moment/timezones/utc/'; // Note trailing slash
 // Error: folders cannot be imported (there is no index.* magic)
 ```
 
+### 2. Imports Field
+
+> **To avoid conflict with `node_modules` packages, the current proposal prefixes all imports with `#name`, so that the fact that an alias is being imported is clear. Whether this restriction is maintained in the final proposal, or what symbol is used is still TBD.**
```

Comment:
I'd prefer if we spelled out the consequence/tradeoff here. Having a prefix will provide a clear signal that it's a "symbolic" specifier but at the same time it will prevent packages that use `imports` from working in any environment (runtime, bundler) that isn't aware of `imports`.

Reply:
Ok, I've added a clarification here, let me know if that works for you.